### DISCUSSION TITLE
在浅色背景下使用看的不是很清晰

### DISCUSSION THREAD

**Discussion by goodenough2021** [yan42685/obsidian-clever-search#20] [General]
![image](https://github.com/yan42685/obsidian-clever-search/assets/92140115/c27e14ed-a486-4fe0-ae37-c56696d01644)
在浅色背景下，字体颜色和背景没有形成反差，看的不是很清楚

**Comment by yan42685** [comment]
审核通过了立马安排浅色主题，现在心情很糟糕写不太动
另外临时解决方案是通过 `style settings` 插件自定义各种颜色

快被官方审核人员给整破防了，要求我做很麻烦的修改，并且给出的理由站不住脚，而且回复极慢 😭 ，我从一月份就开始申请了，合理的修改要求都十分配合。帮忙说句话吧，链接 https://github.com/obsidianmd/obsidian-releases/pull/2958
